0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,252 +1,135 @@
-
-* Proposed -- Nothing has happened yet.
-* Prepared -- Funds are held.
-* Executed -- The transfer has completed.
-* Rejected -- The transfer has been canceled (and funds returned to the sender.)
-
-<span class="show alert alert-info">**Hint:** Authorization holds are the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
-
-The ledger MUST be able to release the held funds to the receiver upon receiving a 32-byte preimage whose SHA-256 hash matches a value provided by the sender. The ledger MUST accept ONLY 32 byte preimages or support specifying the fulfillment length when the transfer is prepared.
-
-The ledger MUST support releasing held funds back to the sender after a timeout.
-
-The ledger MUST support attaching a short message or *memo* to each transfer.
-
-The ledger MUST support notifications to account holders when transfers are prepared, executed, or rejected that affect their accounts.
-
-#### Full Support
-
-The ledger MUST fulfill the requirements for basic support and also the following:
-
-The ledger MUST support memos up to 65535 bytes.
-
-The ledger MUST support sending an authenticated message of up to 65535 bytes to the holder of another account on the ledger.
-
-The ledger MUST support a way to look up a fulfillment by condition hash. It SHOULD automatically reject new transfers (that have not been prepared yet) that have an execution condition for which the ledger already knows the fulfillment. This aids in [error recovery](#error-recovery).
-
-The ledger MUST support preparing, executing, and rejecting transfers in 1 second or less.
-
-The ledger MUST define an [ILP Address](../0015-ilp-addresses/0015-ilp-addresses.md) prefix and scheme such that accounts on the ledger can be addressed using canonical ILP addresses.
-
-It MUST support receivers (and ONLY receivers) rejecting incoming transfers. Receivers MUST be able to submit a rejection message of up to 65535 bytes. The sender of the transfer MUST be notified that the transfer has been rejected and the notification MUST include the rejection message.
-
-### Example Protocols
-
-#### Five Bells Ledger Protocol (5BLP)
-
-Five Bells Ledger Protocol (5BLP) is a RESTful, JSON-based protocol that was developed specifically to provide the minimum functionality required for full Interledger support.
+See [IL-RFC 17](../0017-ledger-requirements/0017-ledger-requirements.md) for a full description of the ledger layer requirements.
 
-A reference implementation of a ledger using 5BLP can be found [here](https://github.com/interledger/five-bells-ledger).
+Most implementations of Interledger use a plugin architecture to abstract the differences between different ledgers. For an example of this, see [IL-RFC 4](../0004-ledger-plugin-interface/0004-ledger-plugin-interface), which defines the interface for the Javascript implementation.
```

Comment:
Broken link: `../0004-ledger-plugin-interface/0004-ledger-plugin-interface` is missing the `.md` on the end